### DISCUSSION TITLE
iPhone和Mac自造词同步出现lost文件夹，导致自造词损坏

### DISCUSSION THREAD

**Discussion by Anber55** [rime/home#1470] [Q&A]
请问各位大佬，我是使用雾凇拼音的全拼用户，用Rime也好几年了，之前同步自造词一直正常，但近一年以来，无论是同步还是部署，rime_ice.userdb或其他自造词文件夹都有可能出现lost文件夹

Mac上的表现是：一旦要出现自造词，就会在一段时间内，疯狂出现自动部署的情况，会导致出现rime_ice.userdb.old文件夹的情况，自造词损坏

iPhone上的表现是：点同步或部署之后在rime_ice.userdb产生lost文件夹

求各位大佬或者有类似情况的支个招，可能是哪儿出了问题，真的是太折腾了，感激不尽
![IMG_0266](https://github.com/rime/home/assets/117334130/1b56574e-27c3-4a3c-9c07-c9e4104fa6f7)
![IMG_0265](https://github.com/rime/home/assets/117334130/25690ec7-bd64-4169-9fcc-8a0ddb51226c)

**Comment by lotem** [comment]
同步文件夹怎样设置的呢？
`installation.yaml` 各个设备上是什么内容？

**Reply by Anber55** [reply]
不好意思，坐火车没网，麻烦大佬帮我看看，感谢
![IMG_0270](https://github.com/rime/home/assets/117334130/e2f477e4-ed3e-466f-8f86-e58dbda17b45)
![IMG_0269](https://github.com/rime/home/assets/117334130/cd12fd6e-9359-472b-96d4-4f6c25643f17)
![IMG_0268](https://github.com/rime/home/assets/117334130/0af2572f-4af5-4241-ad55-4ae18f34506f)
![IMG_0267](https://github.com/rime/home/assets/117334130/06ea2d32-b5cb-49c3-b0c7-6a6270464a19)
<img width="459" alt="截屏2024-02-06 08 54 27" src="https://github.com/rime/home/assets/117334130/eb0c253a-57d0-451e-8a3a-03ec215990c4">

**Reply by Anber55** [reply]
另外，我之前是利用iCloud同步自造词和其他文件，因为iCloud很不稳定，现在是利用Git（配合快捷指令）做同步

**Reply by Anber55** [reply]
麻烦大佬看到回复一下，可能是什么原因，感恩

**Reply by Anber55** [reply]
大佬，麻烦在的时候能答疑一下吗？这个问题困扰我很久了，感激不尽

**Comment by CBAnother** [comment]
@Anber55  你好题主，我想问一下 iPhone 上你用的什么应用程式，我只搜到了一个 iRIME，是辜伟雄开发的

**Reply by Anber55** [reply]
Hamster（仓输入法）很成熟了，推荐

**Reply by CBAnother** [reply]
十分感谢
